examples/step-76/step-76.cc

```diff
@@ -510,7 +510,6 @@ namespace Euler_DG
   EulerOperator<dim, degree, n_points_1d>::EulerOperator(TimerOutput &timer)
     : timer(timer)
   {
-#if DEAL_II_MPI_VERSION_GTE(3, 0)
```

Comment:
This causes the bug. Can you undo all changes in step-76? We can address this one separately.